coderd/database/migrations/000351_add_icon_and_description_template_version_presets.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE template_version_presets
+	ADD COLUMN IF NOT EXISTS description VARCHAR(128) NOT NULL DEFAULT '',
+	ADD COLUMN IF NOT EXISTS icon VARCHAR(256) NOT NULL DEFAULT '';
```

Comment:
Applying the same size constraints as:
* `templates` table: https://github.com/coder/coder/blob/main/coderd/database/dump.sql#L1727-L1730
* `terraform-provider-coder`: https://github.com/coder/terraform-provider-coder/blob/e6d58d0ed91f1bcd614ecbac2db228b64c1de3d4/provider/workspace_preset.go#L98-L115

Reply:
Please add a comment on these fields.

Reply:
Addressed in: https://github.com/coder/coder/pull/18977/commits/dcc661982167936cc8fd7072ba367c64335a08de